install.sh

```
@@ -256,186 +302,362 @@ case "\$1" in
+    create_user_command "cr"
+
+    # 配置 PATH
+    configure_user_path "$user_bin_dir"
+
+    echo -e "${GREEN}✓ 用户命令设置完成${NC}"
+    echo -e "${BLUE}  可用命令: cr, codereview-cli, coderocket${NC}"
+}
+
+# 检测用户的 shell 类型
+detect_user_shell() {
+    local shell_name=$(basename "$SHELL")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)